crates/test_helpers/src/subnormals.rs

```diff
@@ -0,0 +1,39 @@
+pub trait FlushSubnormals: Sized {
+    fn flush(self) -> Self {
+        self
+    }
+}
+
+impl<T> FlushSubnormals for *const T {}
+impl<T> FlushSubnormals for *mut T {}
+
+macro_rules! impl_float {
+    { $($ty:ty),* } => {
+        $(
+        impl FlushSubnormals for $ty {
+            fn flush(self) -> Self {
+                let is_f32 = core::mem::size_of::<Self>() == 4;
+                let ppc_flush = is_f32 && cfg!(all(target_arch = "powerpc64", target_endian = "big", not(target_feature = "vsx")));
```

Comment:
that's wrong, scalar ppc64 (and SVP64, when that is eventually added) doesn't flush subnormals. you need to check for at least altivec

Reply:
The Rust default ppc64 target includes altivec, but might as well check it explicitly